tests/behat/behat-features/replication-aditions.feature

```diff
@@ -0,0 +1,227 @@
+    And I press "Save and publish"
+    Then I save media to array with title "BEHAT: Image media 1"
+
+    And I visit path "media/add/image" on contentpool
+    And I fill in "name[0][value]" with "BEHAT: Image media 2"
+    When I attach the file "core/profiles/demo_umami/modules/demo_umami_content/default_content/images/chili-sauce-umami.jpg" to "files[field_image_0]"
+    And I press "Save and publish"
+    Then I save media to array with title "BEHAT: Image media 2"
+
+    And I visit path "media/add/image" on contentpool
+    And I fill in "name[0][value]" with "BEHAT: Image media 3"
+    When I attach the file "core/profiles/demo_umami/modules/demo_umami_content/default_content/images/chili-sauce-umami.jpg" to "files[field_image_0]"
+    And I press "Save and publish"
+    Then I save media to array with title "BEHAT: Image media 3"
+
+    And I visit path "media/add/image" on contentpool
+    And I fill in "name[0][value]" with "BEHAT: Image media 4"
+    When I attach the file "core/profiles/demo_umami/modules/demo_umami_content/default_content/images/chili-sauce-umami.jpg" to "files[field_image_0]"
+    And I press "Save and publish"
+    Then I save media to array with title "BEHAT: Image media 4"
+
+  @javascript @replication-aditions-default
+  Scenario: Replication media works
+    # Create article with teaser
+    Given I am logged in to contentpool
+    When I visit path "node/add/article" on contentpool
+    And I fill in "title[0][value]" with "BEHAT: Media"
+    And I fill in "field_seo_title[0][value]" with "test media"
+    And I select "-Cooking" from "edit-field-channel"
+    And I press "field_teaser_media_entity_browser_entity_browser"
+    Then I wait for AJAX to finish
+    Then I wait for ".views-row:nth(0)" in entity browser "image_browser"
+    When I click on "0" of created media in "image_browser"
+    And I click on "#edit-submit" in entity browser "image_browser"
+    And I wait for entity browser "image_browser" to close
+    Then I wait for AJAX to finish
+    Then Save "[name='field_teaser_media[target_id]']" value as currently selected
+    And I select "Published" from "moderation_state[0]"
+    And I press "Save as"
+    And I wait for the page to be loaded
+
+    # Replicate
+    Given I open the satellite
+    When I run drush cppull
+    Then drush output should contain "Content of remote /Contentpool/ has been replicated successfully."
+    And I am logged in as a user with the "administrator" role
+
+    # Check article replicated
+    And I am on "/"
+    Then I should see the text "BEHAT: Media"
+
+    # Check media is replicated
+    And I am on "admin/content/media"
+    Then I should see title of currently selected media
+    And I should get a 200 HTTP response
+
+    # Edit article and change teaser
+    Given I visit path "/" on contentpool
+    And I click "BEHAT: Media"
```

Comment:
Please go to admin/content/media and click edit there. Relying on "/" is fragile. Example can be found at 

https://github.com/drunomics/contentpool-client/pull/47/files#diff-3d0be465f9db2bd17590600ad444de8bR102